ompi/mpi/man/man3/MPI_Bcast.md

```diff
@@ -0,0 +1,141 @@
+
+# NAME
+==================================
+
+**MPI_Bcast, MPI_Ibcast** - Broadcasts a message from the process with
```

Comment:
Good catch on boldfacing from the original `.in` file.  I looked a bunch of system man pages on my machine, and they don't appear to be boldfacing the function name here.  So let's leave it out here.